test/integration/pack/test_pack.py

```diff
@@ -2184,3 +2184,21 @@ def test_net_msg_max_invalid_value(cartridge_cmd, project_without_dependencies,
     rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
     assert rc == 1
     assert error_message in output
+
+
+@pytest.mark.parametrize('pack_format', ['tgz'])
+def test_ignore_tmp_dir_content(cartridge_cmd, project_without_dependencies, pack_format, tmpdir):
+    project = project_without_dependencies
+
+    with open(os.path.join(project.path, "tmp", "tmpfile"), "w") as f:
+        f.write("test text")
+
+    cmd = [cartridge_cmd, "pack", pack_format, project.path]
+
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
```

Comment:
How about `subprocess.run(cmd, cwd=project.path)` since you don't need `output`?

Reply:
Fixed